packages/reports/src/Containers/open-positions.jsx

```diff
@@ -505,31 +553,70 @@ const OpenPositions = ({
             {active_positions.length !== 0 && (
                 <React.Fragment>
                     <DesktopWrapper>
-                        <div className='open-positions__contract-types-selector__container'>
-                            <Dropdown
-                                is_align_text_left
-                                name='contract_types'
-                                list={contract_types}
-                                value={contract_type_value}
-                                onChange={handleChange}
-                            />
+                        <div
+                            className={
+                                is_accumulator_selected
+                                    ? 'open-positions__accumulator-container'
+                                    : 'open-positions__contract-types-selector-container'
+                            }
+                        >
+                            <div className='open-positions__accumulator-container__contract-dropdown'>
+                                <Dropdown
+                                    is_align_text_left
+                                    name='contract_types'
+                                    list={contract_types}
+                                    value={contract_type_value}
+                                    onChange={handleContractChange}
+                                />
+                            </div>
+
```

Comment:
@henry-deriv  this line break seems to be extra ) 
and you have run option+shift+f already to prettify the code, right?